eats_worm/Extractor.py

```diff
@@ -505,6 +562,20 @@ def save_dataframe(self):
         print('Export threads+quant:', csv)
         df = self.results_as_dataframe()
         df.to_csv(csv, float_format='%6g')
+        csv_blobs = os.path.join(self.output_dir, 'df_peaks_detected.csv')
+        print('Export blobs: ', csv_blobs)
+        blobs = df.loc[df['T']==0]
+        newcols = ['X', 'Y', 'Z', 'prov', 'ID']
+        blobs = blobs[newcols]
+        blobs = blobs.replace('infilled', 'detected')
+        blobs = blobs.fillna("")
+        blobs = blobs.astype({'X':'int32', 'Y':'int32', 'Z':'int32'})
+        blobs['Status'] = 0
+        blobs['ID'] = blobs['ID'].astype('string')
+        blobs['ID'] = ""
+        print(blobs['ID'].unique())
+        print(blobs['ID'].dtype)
+        blobs.to_csv(csv_blobs, na_rep='')
```

Comment:
The first csv is the spools.csv output from the eats-worm extractor. I think this is what is used in the Curator class. The NPEX peak finding algorithm outputs this df_peaks_detected.csv file so I'm creating a file with that same format as output from the extractor. The alternative to doing this would be to change NPEX to read and edit the spools.csv file but that's pretty much just doing these same lines of code just on the NPEX side.